datastore/integration_test.go

```diff
@@ -120,7 +120,7 @@ func testMain(m *testing.M) int {
 	databaseIDs := []string{DefaultDatabaseID}
 	databasesStr, ok := os.LookupEnv(envDatabases)
 	if ok {
-		databaseIDs = append(databaseIDs, strings.Split(databasesStr, ",")...)
+		databaseIDs = strings.Split(databasesStr, ",")
```

Comment:
Removing the append removes the default database from the list.

Reply:
Oops. This was not intended to be committed; I had done this because testing in the default database wasn't easy without polluting a ton of stuff in existing projects here. Nice catch.